package.json

```diff
@@ -17,6 +17,7 @@
     "prettier": "prettier --ignore-path .prettierignore \"./**/*.{js,ts,tsx,json}\"",
     "format": "yarn lint:fix && yarn prettier --write",
     "format:verify": "yarn prettier --check",
+    "compile": "yarn verify",
```

Comment:
why are we adding this here, if it's the same thing as verify? 

similar situation with verify-all

Reply:
we call it compile in most other repos at this point, just wanted to be consistent